src/guessTheNumber.js

```diff
@@ -0,0 +1,16 @@
+const randomNum = Math.floor(Math.random() * 100)
+export const guessTheNumber = (input) => {
+  if(Number.isInteger(input) && input < randomNum) {
+    return 'Your guess is kinda low'
+  } else if(Number.isInteger(input) && input > randomNum) {
```

Comment:
Hi @aaronev thank you for the input. I was using best practices from MDN. I mirrored their structure when using conditionals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else